### PR TITLE
#1647 TimerLab error when no slides selected or empty ppt

### LINKED ISSUE #1647: Errors when there are no slides at all in PPT 2016
How to reproduce:
Just delete all the slides in the presentation and click on Timer Lab or Picture Slides Lab etc.
image

Any code that relies on ` PowerPointSlide currentSlide = this.GetCurrentSlide();` will return errors.

### PR DESCRIPTION
**Pull request by LiXiaoooowei**
Fixes #1647

Tested in PPT2016.

Changes:

- Greyed out TimerLab ribbon button when no slide is selected or ppt is empty.

- Remove TimerPane when TimerLab is previously opened but no slides selected later.

### CODE REVIEW

File: PowerPointLabs/PowerPointLabs/TimerLab/NotesToTimer.cs [hunk 1]
```diff
@@ -0,0 +1,9 @@
+﻿namespace PowerPointLabs.TimerLab
+{
+    internal static class NotesToTimer
```
Comment (initialshl) [inline PowerPointLabs/PowerPointLabs/TimerLab/NotesToTimer.cs]: May I know why this class is called NotesToTimer? Or is there any better place to put IsTimerEnabled?

**Comment by leeyh20** [comment]
@LiXiaoooowei Could this be tested on PPT2013?

File: PowerPointLabs/PowerPointLabs/TimerLab/NotesToTimer.cs (hunk 1 above)
Comment (leeyh20) [reply PowerPointLabs/PowerPointLabs/TimerLab/NotesToTimer.cs]: @LiXiaoooowei Would you be updating this PR?

**Comment by blewjy** [comment]
Tested on PPT2013. Working as described!

File: PowerPointLabs/PowerPointLabs/TimerLab/NotesToTimer.cs (hunk 1 above)
Comment (LiXiaoooowei) [reply PowerPointLabs/PowerPointLabs/TimerLab/NotesToTimer.cs]: @initialshl I am trying to follow the names set by other labs for similar functionalities, i.e. `NotesToAudio` and `NotesToCaption`.

Comment (initialshl) [reply PowerPointLabs/PowerPointLabs/TimerLab/NotesToTimer.cs]: Hmm I think that the `NotesToAudio` and `NotesToCaption` refers to the slide's speaker's notes (text-to-speech). You may want to change it to just `TimerLab.cs`

Comment (LiXiaoooowei) [reply PowerPointLabs/PowerPointLabs/TimerLab/NotesToTimer.cs]: @initialshl Code edited according to suggestion.

**Pull request merged**